utilities/src/main/java/org/craftercms/commons/config/ConfigUtils.java

```diff
@@ -63,15 +71,25 @@
      *
      * @throws ConfigurationException if an error occurs while reading the configuration
      */
-    public static HierarchicalConfiguration<ImmutableNode> readXmlConfiguration(InputStream input, String fileEncoding)
+    public static HierarchicalConfiguration<ImmutableNode> readXmlConfiguration(InputStream input,
```

Comment:
Shouldn't it be better to get rid of this method which is almost a duplicate of the one below, and in this case pass a null `Character`?

Reply:
I didn't replace it because of the use of `FileHandler`, the method was used in another class and I move it here so I don't know if just one will work for both cases

Reply:
Ah yes I see, it's ok then.